site/content/en/contribute/howitworks/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Writing Code"
+linkTitle: "Writing Code"
+type: docs
+weight: 40
+description: >
+    How to modify Kustomize
+---
+
+{{< alert color="success" title="Info" >}}
+Kustomize supports go workspace mode, so if adding locally modified modules, remember to also add it to the [go.work](https://github.com/kubernetes-sigs/kustomize/blob/master/go.work) file.
+
+{{< /alert >}}
+
+## Call stack
+Call stack when running `kustomize build`, with links to code:
```

Comment:
this part changed quite a bit. I updated all the links, but since the code changed quite a bit, I'd like some confirmation on what to display here. Also, not quite sure if we want to capture every single call... 

for reference, this is what we had before: https://kubectl.docs.kubernetes.io/contributing/kustomize/howitworks/

alternatively, we can just mention the main functions: `Build Command`([the entry point](https://github.com/kubernetes-sigs/kustomize/blob/master/kustomize/commands/build/build.go#L65)), [MakeKustomizer](https://github.com/kubernetes-sigs/kustomize/blob/master/api/krusty/kustomizer.go#L36), [Run](https://github.com/kubernetes-sigs/kustomize/blob/master/api/krusty/kustomizer.go#L53), and the bit that writes the output to a yaml file.

what do you think? I'm not sure it is so helpful linking to every single func and the additional descriptions, since we can read them once we navigate through the code. 

Once one identifies the entry point and the main functions, it's easy to navigate on Github, by clicking through. I wouldn't go back to this doc in order to do that.

At least, I found it a bit confusing when following the instructions

@KnVerey

Reply:
addressed here: https://github.com/kubernetes-sigs/kustomize/pull/4709/commits/b42597ec29f73f8976a8cd5273134baa32d04aa2